app/models/concerns/mark_as_concern.rb

```diff
@@ -8,7 +8,7 @@ module MarkAsConcern
     before_save :ensure_unique_most_request, if: :is_most_request?
   end
 
-  MARK_AS_LIST = %w(gender user_visit location ticket_tracking service_accessed)
+  MARK_AS_LIST = %w(gender user_visit location ticket_tracking service_accessed feedback_like feedback_dislike feedback_rating)
```

Comment:
@radinreth please ensure that this is not activate to Web UI as we didn't allow them to mark in UI yet

Reply:
currently, we don't have any UI to configure these variables yet, setting will be done via terminal.